comid-code-points.cddl

```diff
@@ -89,8 +83,8 @@ comid.element-value = 1
 comid.reference-claims = 0
 comid.endorsements = 1
 comid.identity-claims = 2
-comid.ref-instance-claims = 3
-comid.endors-instance-claims = 4
+comid.reference-instance-claims = 3
+comid.endorsements-instance-claims = 4
```

Comment:
maybe s/endorsements/endorsed/ ?  (and change L84 as well)

Reply:
No as I said this is to maintain consistency, will retain like this for now to be consistent with the rest of the variables...

Reply:
np.  I've opened #58 to track this

Reply:
Thanks, we can change the consistency aspect globally on Issue #58. For now we can proceed with this change!